05-UI - ReactJS/my-app/src/App.js

```diff
@@ -0,0 +1,115 @@
+//import React
+import React, { Component } from 'react';
+import logo from './logo.svg';
+
+// components
+import CreateForm from './components/CreateForm';
+import FavoriteMovies from './components/FavoriteMovies';
+import EditMovie from './components/EditMovie';
+
+// styles
+import './App.css';
+import './movies.css';
+import './CreateForm.css';
+
+//import Json of Movies
+import { movies } from './movies.json';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movies,
+      id: null,
+      showModal: false,
+      movie : null
+    };
+    this.handleAddMovie = this.handleAddMovie.bind(this);
+    this._handleEditing = this._handleEditing.bind(this);
+    this.ModalView = this.ModalView.bind(this);
+    this.MovieEdit = this.MovieEdit.bind(this);
+    this.RemoveMovie = this.RemoveMovie.bind(this);
+
+  }
+
+  handleAddMovie(movie) {
+    this.setState({
+      movies : [...this.state.movies, movie]
+    })
+  }
+  _handleEditing(idMovie) {
+    let data
+    this.state.movies.forEach(movie => {
```

Comment:
Acá podrías haber usado un `this.state.movies.filter()` para quedarte con el elemento